frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -1804,21 +1804,23 @@ func pathRelativeToWorkingDir(s llb.State, p string, platform ocispecs.Platform)
 		return "", err
 	}
 
-	p, err = system.CheckSystemDriveAndRemoveDriveLetter(p, platform.OS)
+	keepSlash := true
```

Comment:
This doesn't seem to be a settable variable, so you can just pass `true` directly.

Reply:
Ok, can do that, thx.